docs/modules/installation/pages/requirements.adoc

```diff
@@ -1,17 +1,17 @@
 = Requirements
 
-== Runtime Requirements
+== Runtime requirements
 
 * http://docs.grafana.org/installation[Grafana]
-{page-component-title} works with Grafana Server version *7.0* and up.
-It has been tested with Grafana versions up to 7.2 at the time of release.
+{page-component-title} works with Grafana Server version *8.0* and up.
```

Comment:
Is there a way we could use a variable here?

Reply:
Probably.  Where are we defining variables for Helm docs?